### ISSUE TITLE
[Accessibility] Unable to read search box when results combo box is displayed

### ISSUE THREAD

**Issue by rafa11rodrigues** [microsoft/vscode#166072]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.73.1 (and all previous ones)
- OS Version: Windows 11 and 10, both desktop and web version

Steps to Reproduce:

1. Open VS Code while running any screen reader (I use JAWS)
2. Open any popup which displays a set of results based on a search box (e.g. Ctrl+Shift+P for Command Palette, Ctrl+P to open file, or click in language mode at status bar to change language)
3. Type some text to see results
4. Try to move left and right to read the text box, or backspace to delete something
5. Try to tab to move back to the search box

When any result is shown, it's impossible to edit the search term for a blind person, because the combo box steals the focus. I'd like to edit the text, read by letter and word, delete...
Maybe a good fix would be allowing tab to move focus from search box to results list and vice-versa in a circular way.

Just to be clear: editing is possible, the cursor moves left and right, I can select text, delete, whatever I want. The problem is the feedback for a blind person using an screen reader.

**Comment by isidorn** [comment]
@rafa11rodrigues can you reproduce this with NVDA?
JAWS has some limitations with VS Code.

**Comment by rafa11rodrigues** [comment]
@isidorn Yes, the same behavior with NVDA.
Good to know about these JAWS specific limitations... is there a list of them somewhere?

**Comment by isidorn** [comment]
Aha you are right. I can also reproduce. We currently keep focus in the QuickPick input box but we used activedescendant to point the screen reader to the below elements when users navigates there.
@leonardder is there something better we could do on the vscode side? So that NVDA would still read the QuickPick input even when activedescendant focus is on one of it's child elements?

@rafa11rodrigues here's the issue with more details for JAWS https://github.com/microsoft/vscode/issues/95225

**Comment by jvesouza** [comment]
It's possible that I misunderstood the problem, but the fact is that I couldn't reproduce it using orca.
I used the following steps:

1. I pressed ctrl+shift+p
2. I typed focus
3. I pressed down arrow and up arrow a few times
4. I pressed left arrow and right arrow a few times
5. I edited the search field

Results:
Every time I press down arrow or up arrow the selected command was announced by orca.
Likewise, whenever I pressed left or right arrow, the focused character in the search field was read by orca.
Edits made in the search field were also reported correctly.

**Comment by isidorn** [comment]
@jvesouza thanks for confirming that this works well on Orca. 
Ideally other screen readers use that as inspiration to implement the same behaviour in other screen readers :)

**Comment by TylerLeonhardt** [comment]
@isidorn what do we typically do when behavior works in one screen reader and not the other? And does the answer matter which screen reader works and which doesn't?

**Comment by isidorn** [comment]
@TylerLeonhardt we usually try to talk to the authors of the screen reader that does not work and see if there is something they can do to fix this.

In this case I do not think there is anything we can fix on our side. So I will go ahead and mark this bug as upstream
@rafa11rodrigues could you open this issue on the NVDA side here https://github.com/nvaccess/nvda

**Comment by VSCodeTriageBot** [comment]
We closed this issue because we [don't plan to address it](https://aka.ms/vscode-out-of-scope) in the foreseeable future. If you disagree and feel that this issue is crucial: we are happy to listen and to reconsider.

If you wonder what we are up to, please see our [roadmap](https://aka.ms/vscoderoadmap) and [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Thanks for your understanding, and happy coding!
